README.md

```diff
@@ -38,6 +38,8 @@
     <li>Git</li>
     <li>GitHub</li>
     <li>Jest</li>
+    <li>React</li>
+
   </ul>
 ### Tech Stack <a name="tech-stack"></a>
 
```

Comment:
- [ ] @otmanTR great work in your readme, it would be nice if you customized your MIT license, also add content to your `Show your support` section, also check on your future feature section the format isn't the same as in other section, kindly rectify it.

Reply:
@danmainah hello, could you please explain to me how can I customize my MIT license? My name is in MIT.md file. I don't know another way to customize MIT licence.